### PR TITLE
Wallet, Explorer and Faucet Services

### PR DESCRIPTION
**Pull request by fboucquez**
Added the first version of the explorer.  This version relies on the https://github.com/nemgrouplimited/symbol-explorer/pull/685 fix that allows custom configuration. 

The PR is using one a fboucquez explorer image with the image from the fix above. Once the explorer PR is merged, I would go back to use the official symbolplatform docker image. 

@segfaultxavi @Wayonb , if you want to try it out please checkout this branch and "npm -g install" it to run from source code.

1 explorer has been added to the default `bootstrap` preset

### CODE REVIEW

**Comment by segfaultxavi** [comment]
I checked out the `explorer` branch and run `symbol-bootstrap start -p bootstrap`. I can access the API node, the explorer and the wallet (at ports 3000, 80 and 90).
But explorer and wallet cannot connect to the testnet, they say "Node is not available. Please check your node or network settings.".

**Comment by fboucquez** [comment]
They won't connect. It's a completely different network. Different
generation hash and configuration. Xym doesn't even exist in this network.

**Comment by segfaultxavi** [comment]
I'll try to be clearer. I run `symbol-bootstrap start` and go to `localhost:90`. I can see the Wallet, but it does not work because there is an orange box saying: Node is not available. Please check your node or network settings.

What can I do?

**Comment by fboucquez** [comment]
Added faucet and wallet web.

The wallet web hasn't been confirmed. We may remove it.

File: config/wallet/profileImporter.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    window.onload = function () {
+      //save entered gmail address
+      document.getElementById('save').addEventListener(
+        'click',
+        function () {
+          try {
+            const profile = JSON.parse(document.getElementById('profile').value)
+            localStorage.setItem('profiles', JSON.stringify(profile.profiles))
+            localStorage.setItem('accounts', JSON.stringify(profile.accounts))
+            alert('Profile have been saved')
```
Comment (segfaultxavi) [inline config/wallet/profileImporter.html:31]:
```suggestion
            alert('Profile has been saved')
```

File: config/wallet/profileImporter.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            console.log('Profile have been saved')
```
Comment (segfaultxavi) [inline config/wallet/profileImporter.html:32]:
```suggestion
            console.log('Profile has been saved')
```

File: config/wallet/profileImporter.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+          } catch (e) {
+            alert('Profile could not be saved been saved ' + e)
```
Comment (segfaultxavi) [inline config/wallet/profileImporter.html:34]:
```suggestion
            alert('Profile could not be saved' + e)
```

File: config/wallet/profileImporter.html [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+            console.error('Profile could not be saved been saved', e)
```
Comment (segfaultxavi) [inline config/wallet/profileImporter.html:35]:
```suggestion
            console.error('Profile could not be saved', e)
```

**Review by segfaultxavi — CHANGES_REQUESTED** [review]
Typos

File: config/wallet/profileImporter.html [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+  <script type="application/javascript">
+    window.onload = function () {
+      //save entered gmail address
+      document.getElementById('save').addEventListener(
+        'click',
+        function () {
+          try {
+            const profile = JSON.parse(document.getElementById('profile').value)
+            localStorage.setItem('profiles', JSON.stringify(profile.profiles))
+            localStorage.setItem('accounts', JSON.stringify(profile.accounts))
```
Comment (segfaultxavi) [inline config/wallet/profileImporter.html:30]: Could you also save (and restore) the `mosaicConfiguration` setting?
It stores whether each mosaic is shown in each account's assets list, and it's helpful when we're not interested in some of the mosaics.

File: config/wallet/profileImporter.html (hunk 2 above)
Comment (fboucquez) [reply config/wallet/profileImporter.html]: Fixed

File: config/wallet/profileImporter.html (hunk 3 above)
Comment (fboucquez) [reply config/wallet/profileImporter.html]: Fixed

File: config/wallet/profileImporter.html (hunk 4 above)
Comment (fboucquez) [reply config/wallet/profileImporter.html]: Fixed

File: config/wallet/profileImporter.html (hunk 5 above)
Comment (fboucquez) [reply config/wallet/profileImporter.html:30]: Added

**Review by segfaultxavi — APPROVED** [review]
My concerns have been addressed.

**Comment by fboucquez** [comment]
Right now I'm waiting for an official faucet alpha image in order to merge the PR:

https://github.com/nemgrouplimited/symbol-faucet/issues/49

And then, I would need released versions of all the Images and TS SDK in order to release.

File: CHANGELOG.md [hunk 6]
```diff
@@ -12,6 +12,9 @@ The changelog format is based on [Keep a Changelog](https://keepachangelog.com/e
 ---|---|---
 Symbol Bootstrap | v0.3.0 | [symbol-bootstrap](https://www.npmjs.com/package/symbol-bootstrap)
 
+- **New Service:** `Walelt`. Bootstrap private network starts a Wallet service in [http://localhost:80/](http://localhost:90/) when using `--assembly full`. . **Warning:** This wallet service is for demonstration proposes.
```
Comment (segfaultxavi) [inline CHANGELOG.md:15]:
```suggestion
- **New Service:** `Wallet`. Bootstrap private network starts a Wallet service in [http://localhost:80/](http://localhost:90/) when using `--assembly full`. . **Warning:** This wallet service is for demonstration purposes only.
```

Port 80 or 90!?!?!?

File: CHANGELOG.md [hunk 7]
```diff
@@ -12,6 +12,9 @@ The changelog format is based on [Keep a Changelog](https://keepachangelog.com/e
...
+- **New Service:** `Explorer`. Bootstrap private network starts a Faucet service in [http://localhost:90/](http://localhost:80/) when using `--assembly full`.
```
Comment (segfaultxavi) [inline CHANGELOG.md:16]: Port 90 or 80?

File: CHANGELOG.md [hunk 8]
```diff
@@ -12,6 +12,9 @@ The changelog format is based on [Keep a Changelog](https://keepachangelog.com/e
 ---|---|---
 Symbol Bootstrap | v0.3.0 | [symbol-bootstrap](https://www.npmjs.com/package/symbol-bootstrap)
 
+- **New Service:** `Walelt`. Bootstrap private network starts a Wallet service in [http://localhost:80/](http://localhost:90/) when using `--assembly full`. . **Warning:** This wallet service is for demonstration proposes.
+- **New Service:** `Explorer`. Bootstrap private network starts a Faucet service in [http://localhost:90/](http://localhost:80/) when using `--assembly full`. 
+- **New Service:** `Faucet`. Bootstrap private network starts an Explorer service in [http://localhost:100/](http://localhost:100/) when using `--assembly full`.
```
Comment (segfaultxavi) [inline CHANGELOG.md:17]:
```suggestion
- **New Service:** `Faucet`. Bootstrap private network starts a Faucet service in [http://localhost:100/](http://localhost:100/) when using `--assembly full`. 
```

File: README.md [hunk 9]
```diff
@@ -64,8 +64,9 @@ Properties in each file override the previous values (by object deep merge).
...
--   `-p bootstrap -a light`: A [light](https://github.com/nemtech/symbol-bootstrap/blob/main/presets/bootstrap/assembly-light.yml) network. It's a version of bootstrap with 1 mongo database, 1 dual peer and 1 rest gateway. Great for faster light e2e automatic testing. Nemesis block is generated.
+-   `-p bootstrap`: Default [preset](https://github.com/nemtech/symbol-bootstrap/blob/main/presets/bootstrap/network.yml). It's a private network with 1 mongo database, 2 peers, 1 api, 1 rest gateway, 1 faucet, 1 explorer and 1 wallet. Nemesis block is generated.
+-   `-p bootstrap -a light`: A [light](https://github.com/nemtech/symbol-bootstrap/blob/main/presets/bootstrap/assembly-light.yml) network. It's a version of bootstrap preset with 1 mongo database, 1 dual peer and 1 rest gateway. Great for faster light e2e automatic testing. Nemesis block is generated.
+-   `-p bootstrap -a full`: A [full](https://github.com/nemtech/symbol-bootstrap/blob/main/presets/bootstrap/assembly-full.yml) network. The default bootstrap preset plus 1 wallet, 1 explorer and 1 faucet. Great for demonstration purposes. Nemesis block is generated.
```
Comment (segfaultxavi) [inline README.md:69]: Two lines above it says that the default preset ALREADY has a Faucet, an Explorer and a Wallet. Which one is correct?

**Review by segfaultxavi — CHANGES_REQUESTED** [review]
Typos and some clarification needed.

File: CHANGELOG.md (hunk 7 above)
Comment (fboucquez) [reply CHANGELOG.md]: Fixed, 80 is the wallet 90 the explorer

File: CHANGELOG.md (hunk 8 above)
Comment (fboucquez) [reply CHANGELOG.md]: Fixed

File: README.md (hunk 9 above)
Comment (fboucquez) [reply README.md:69]: Thanks! default bootstrap load the same services, full will load wallet, explorer and faucet

File: CHANGELOG.md [hunk 10]
```diff
@@ -12,6 +12,9 @@ The changelog format is based on [Keep a Changelog](https://keepachangelog.com/e
 ---|---|---
 Symbol Bootstrap | v0.3.0 | [symbol-bootstrap](https://www.npmjs.com/package/symbol-bootstrap)
 
+- **New Service:** `Wallet`. Bootstrap private network starts a Wallet service in [http://localhost:80/](http://localhost:80/) when using `--assembly full`. . **Warning:** This wallet service is for demonstration proposes only.
```
Comment (segfaultxavi) [inline CHANGELOG.md:15]:
```suggestion
- **New Service:** `Wallet`. Bootstrap private network starts a Wallet service in [http://localhost:80/](http://localhost:80/) when using `--assembly full`. . **Warning:** This wallet service is for demonstration purposes only.
```

Comment (segfaultxavi) [reply CHANGELOG.md:15]: Be careful! You missed one! :D

**Review by segfaultxavi — CHANGES_REQUESTED** [review]

**Pull request merged**